github-repo-manager/src/main/java/com/axonivy/github/GitHubRepos.java

```diff
@@ -41,6 +41,9 @@ public static List<String> repos(String version) {
           "process-editor-client",
           "process-editor-core",
           "inscription-client",
+          "config-editor-client",
+          "form-editor-client",
+          "ui-components",
```

Comment:
@ivy-edp is the new variable editor also covered? is this the config-editor-client?

Reply:
yes the variable editor is in the config-editor-client repo

Reply:
@alexsuter Yes, as far as i can tell and see, the variable editor lives in the config-editor-client repo and is therefore covered. @ivy-lli right?